application/language/german/general_words_lang.php

```diff
@@ -19,18 +30,66 @@
 $lang['general_word_confirmed'] = 'Bestätigt';
 $lang['general_word_needed'] = 'Benötigt';
 
+$lang['general_word_no'] = 'Nein';
+$lang['general_word_yes'] = 'Ja';
+$lang['general_word_method'] = 'Methode';
+
 $lang['general_word_sent'] = 'Gesendet';
 $lang['general_word_received'] = 'Erhalten';
 $lang['general_word_requested'] = 'Beantragt';
```

Comment:
"Angefordert" is a better translation for "requested"

Reply:
ok